### PR TITLE
bundle the language directory

### PR DESCRIPTION
**Pull request by figsoda**
this introduces the environment varialbe  `TOPIARY_LANGUAGE_DIR` at build time and allows running topiary outside of the directory

this will make it easier to be packaged in nixpkgs and also allows `nix run`ing without cloning the repository

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -69,7 +70,8 @@ fn run() -> FormatterResult<()> {
     let query_path = if let Some(query) = args.query {
         query
     } else if let Some(language) = args.language {
-        str::to_lowercase(format!("languages/{language:?}.scm").as_str())
+        PathBuf::from(option_env!("TOPIARY_LANGUAGE_DIR").unwrap_or("language"))
```
Comment (torhovland) [inline src/main.rs:73]: Should be `unwrap_or("languages")`, right?

**Review by torhovland — CHANGES_REQUESTED** [review]
Nice PR!

File: src/main.rs (hunk 1 above)
Comment (figsoda) [reply src/main.rs]: fixed it

**Review by torhovland — APPROVED** [review]

**Comment by torhovland** [comment]
Thanks!

**Pull request merged**